src/System.ServiceProcess.ServiceController/src/ApiCompatBaseline.net46.txt

```diff
@@ -0,0 +1,3 @@
+MembersMustExist : Member 'System.ServiceProcess.ServiceController.StartType.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
The project is configured to still target net46 and the errors that apicompat shows up are correct. I can remove the baseline and upgrade the targetinpack to net461 if that is the right thing to do? @ericstj , @pallavit